cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -405,6 +419,7 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
         private String zkLoggingHash = "";
         private String kafkaLoggingHash = "";
         private String kafkaBrokerConfigurationHash = "";
+        private ConfigMap kafkaCm;
```

Comment:
I still don't understand why do you need to go configuration -> config map -> configuration instead of just using the configuration. Also, all the methods refer to it as `AncillaryCm`, so it should be probably reflected here as well.